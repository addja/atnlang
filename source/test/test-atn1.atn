atn a1 {
	node n1 {
		arc (#.0 != ".") goto n1 {
			print #.0, "%n";
		}
		arc (true) goto n2 {
			print #.0, "%n";
		}
	}
	node n2 accept;
}

def main() {
   atn(a1);
}

