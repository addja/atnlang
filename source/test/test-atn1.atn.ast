(PROGRAM (atn a1 (NODELIST (node n1 (ARC_LIST (arc (!= (# 0) ".") (goto n1) (LIST_INSTR (print (# 0) "%n"))) (arc true (goto n2) (LIST_INSTR (print (# 0) "%n"))))) (node n2 accept))) (def main PARAMS (LIST_INSTR (ATNCALL a1))))