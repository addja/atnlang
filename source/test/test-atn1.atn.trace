main() <entry point>
|   a1() <line 14>
|   return true <line 7>
return <line 7>
