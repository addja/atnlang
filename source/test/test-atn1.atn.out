El
gato
corre
a
doce
km
por
hora
el
cuatro_de_marzo_a_las_seis_de_la_tarde
.
